src/deploy/m_coordinator.cpp

```diff
@@ -135,23 +136,38 @@ void Coordinator::do_join_query() {
 
 //		vector<int> uppers;
 		vector<string> mostuppers;
-		mostuppers.push_back("10.11.1.197");
+		mostuppers.push_back("10.11.1.190");
```

Comment:
it's 1-4-4 model, and next we must change it to 1-8-8 and 1-2-2. here 4-4 are on the same nodeset.